services/infinispan-device-registry/src/main/resources/application.yml

```diff
@@ -0,0 +1,43 @@
+hono:
+  app:
+    type: infinispan
```

Comment:
what is this property good for? Can it have another value?

Reply:
I guess this is a legacy from the other device registry (which is configurable with either `dummy` or `file` ). 
But as we will probably package this as a separate image there is no reason to use this.
Maybe later we could use this to choose between the embedded cache or the client/server one ? @dejanb what do you think, should we keep that option open ?

Reply:
We can always add it later if we need it. Right now, as this registry has its own app we don't really need it

Reply:
I removed it